ui-v2/app/styles/components/tabular-collection.scss

```diff
@@ -204,19 +147,3 @@ html.template-instance.template-show main table.exposedpaths tr {
     display: none;
   }
 }
-@media #{$--lt-medium-table} {
-  /* Token > Policies */
-  /* Token > Your Token */
-  html.template-token.template-list tr > :nth-child(2),
-  html.template-token.template-list tr > :nth-child(4),
-  html.template-token.template-list tr th:nth-child(5),
-  html.template-token.template-list main table tr td.me ~ td:nth-of-type(5) {
-    display: none;
-  }
-  html.template-service.template-show #instances tr > :nth-child(3) {
-    display: none;
-  }
-  %instances-row > * {
-    width: calc(100% / 4);
-  }
-}
```

Comment:
Managed to delete quite a lot of the old table centric CSS for service, instance and token listings.